gems/sorbet-runtime/lib/types/private/runtime_levels.rb

```diff
@@ -53,10 +53,29 @@ def self.default_checked_level=(default_checked_level)
     if @has_read_default_checked_level
       raise "Set the default checked level earlier. There are already some methods whose sig blocks have evaluated which would not be affected by the new default."
     end
+    if !LEVELS.include?(default_checked_level)
+      raise "Invalid `checked` level '#{default_checked_level}'. Use one of: #{LEVELS}."
+    end
+
     @default_checked_level = default_checked_level
   end
 
   def self._toggle_checking_tests(checked)
     @check_tests = checked
   end
+
+  private_class_method def self.set_enable_checking_in_tests_from_environment
+    if ENV['SORBET_RUNTIME_ENABLE_CHECKING_IN_TESTS']
+      enable_checking_in_tests
+    end
+  end
+  set_enable_checking_in_tests_from_environment
+
+  private_class_method def self.set_default_checked_level_from_environment
+    level = ENV['SORBET_RUNTIME_DEFAULT_CHECKED_LEVEL']
+    if level
+      self.default_checked_level = level.to_sym
+    end
+  end
+  set_default_checked_level_from_environment
```

Comment:
This strikes me as a roundabout way to set a default.

Why not handle this earlier on line 24, where the default is actually set?

```ruby
@default_checked_level = :always
```

changing this to something like

```ruby
@default_checked_level =
  if (level = ENV['SORBET_RUNTIME_DEFAULT_CHECKED_LEVEL'])
    raise "..." unless LEVELS.include?(level.to_sym)

    level.to_sym
  else
    :always
  end
```

Reply:
Yea, it's definitely a little roundabout. I made the decision for a few reasons:
- Consolidate the `LEVELS` validation to a single place - the `default_checked_level=` method.
- Make it possible to write a test for this functionality changes.
- I initially implemented like you suggested but then that `level` local variable just exists at the class level which feels weird. Once it wasn't a simple one-liner I felt it made more sense in a method.